inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -605,6 +608,50 @@ descriptive(study_data_cleaned,
   kable(digits = 2)
 ```
 
+```{r visualise_population_flow}
```

Comment:
@zkamvar when you have time can you try and work your magic on this please? its not agreeing with me completely...

Reply:
would be nice i think to have the text either side of the boxes maybe ? 
Got it from the [fundamentals of data visualisation book](https://github.com/clauswilke/dataviz/blob/master/nested_proportions.Rmd)

Reply:
Unfortunately, having the text on either side of the diagram (instead of centered in the bars) is a lot of work :sweat: